tests/pr_test.go

```diff
@@ -98,78 +110,62 @@ func TestInstancesInSchematics(t *testing.T) {
 		{Name: "cos_instance_access_tags", Value: permanentResources["accessTags"], DataType: "list(string)"},
 		{Name: "at_cos_bucket_access_tags", Value: permanentResources["accessTags"], DataType: "list(string)"},
 		{Name: "cloud_log_data_bucket_access_tag", Value: permanentResources["accessTags"], DataType: "list(string)"},
-		{Name: "prefix", Value: options.Prefix, DataType: "string"},
+		{Name: "kms_endpoint_type", Value: "public", DataType: "string"},
+		{Name: "provider_visibility", Value: "public", DataType: "string"},
+		{Name: "management_endpoint_type_for_bucket", Value: "public", DataType: "string"},
+		{Name: "cloud_logs_policies", Value: cloud_logs_policies, DataType: "list(object)"},
```

Comment:
Why are we setting this to public? Previously it was set to public in upgrade test only because that was not Schematics test. Setting this to public for Schematics test - is missing the entire concept.